build.gradle

```diff
@@ -122,6 +128,17 @@ def signingKey(project) {
       }
     } // signingConfigs
 
+    playAccountConfigs {
+      defaultAccountConfig {
+        serviceAccountEmail = 'name@server.com'
+        pk12File = project.file('key.p12') // Will need to encrypt this in Travis, obvs
```

Comment:
@mvl22 sure.  This is about providing API credentials for our CI builds (running in Travis) to publish beta artifacts to Google Play.  You could think of it like the API key that's used for hitting the CycleStreets APIs.

I tried to set this up by going to the Developers Console (http://play.google.com/apps/publish) and clicked Settings, then API Access.  I got this screen:
![screen shot 2017-06-05 at 08 33 16](https://cloud.githubusercontent.com/assets/6017680/26774756/b25d619c-49c9-11e7-824b-4892c9c89fae.png)

I suspect if you go to the same place as the `info@cyclestreets.net` account, you'd get the option to create/enable an API key.  You could then privately send that `.p12` file to me, and I can do the rest.

Does that give you enough to go on?